config_proc_macro/src/attrs.rs

```diff
@@ -27,6 +29,11 @@ pub fn find_config_value(attrs: &[syn::Attribute]) -> Option<String> {
     attrs.iter().filter_map(config_value).next()
 }
 
+/// Returns `true` if the there is at least one `unstable` attribute in the given slice.
+pub fn find_unstable_variant(attrs: &[syn::Attribute]) -> bool {
+    attrs.iter().any(is_unstable_variant)
+}
```

Comment:
A little bit of bike shedding on the name, but I feel like `has_unstable_variants` or `any_unstable_variants` would better convey that this returns a bool. Not a huge deal so don't feel like you need to change this.

Reply:
Renamed